shared/src/main/scala/uk/gov/homeoffice/drt/arrivals/Arrival.scala

```diff
@@ -276,35 +270,33 @@ object Arrival {
   implicit val predictionsRw: ReadWriter[Predictions] = macroRW
   implicit val arrivalRw: ReadWriter[Arrival] = macroRW
   implicit val totalPaxSourceRw: ReadWriter[TotalPaxSource] = macroRW
+  implicit val passengersSourceRw: ReadWriter[Passengers] = macroRW
 
   def apply(Operator: Option[Operator],
-            Status: ArrivalStatus,
-            Estimated: Option[Long],
-            Predictions: Predictions,
-            Actual: Option[Long],
-            EstimatedChox: Option[Long],
-            ActualChox: Option[Long],
-            Gate: Option[String],
-            Stand: Option[String],
-            MaxPax: Option[Int],
-            ActPax: Option[Int],
-            TranPax: Option[Int],
-            RunwayID: Option[String],
-            BaggageReclaimId: Option[String],
-            AirportID: PortCode,
-            Terminal: Terminal,
-            rawICAO: String,
-            rawIATA: String,
-            Origin: PortCode,
-            Scheduled: Long,
-            PcpTime: Option[Long],
-            FeedSources: Set[FeedSource],
-            CarrierScheduled: Option[Long] = None,
-            ApiPax: Option[Int] = None,
```

Comment:
We also need to handle deserialising arrivals with no pax source and an apipax figure - ie add it to the TotalPax map